packages/client-sdk-web/src/storage-client-props.ts

```diff
@@ -0,0 +1,13 @@
+import {CredentialProvider} from '.';
+import {StorageConfiguration} from './config/storage-configuration';
+
+export interface StorageClientProps {
+  /**
+   * Configuration settings for the storage client
+   */
+  configuration?: StorageConfiguration;
```

Comment:
Should this be optional?

Reply:
Yes, if the configuration is not specified, we use the DefaultStorageConfiguration when creating the PreviewStorageClient like below:
```
const configuration =
      props.configuration ?? getDefaultStorageConfiguration();
```

```
function getDefaultStorageConfiguration(): StorageConfiguration {
  return StorageConfigurations.Laptop.latest();
}
```

Reply:
Ok. I don't think we do that in other SDKs so wanted to confirm.